api/volume.go

```diff
@@ -127,6 +129,18 @@ func (s *Server) VolumeCreate(rw http.ResponseWriter, req *http.Request) error {
 		}
 	}
 
+	if volume.AccessMode == "" && volume.FromBackup != "" {
+		_, volumeName, _, err := backupstore.DecodeBackupURL(volume.FromBackup)
+		if err != nil {
+			return fmt.Errorf("failed to get backup and volume through URL: %v", err)
+		}
+		bs, err := s.m.ListBackupsForVolumeSorted(volumeName)
+		if err != nil || bs == nil {
+			return fmt.Errorf("error listing backups for volume '%s': %v", volumeName, err)
```

Comment:
```suggestion
			return errors.Wrapf(err, "failed to list backups for volume %v", volumeName)
```

Reply:
Thank you. I will follow the suggestion to wrap up the error.